students/Craig_Morton/lesson07/html_render.py

```diff
@@ -0,0 +1,130 @@
+# ------------------------------------------------- #
+# Title: Lesson 7, pt 1/2, html render
+# Dev:   Craig Morton
+# Date:  9/14/2018
+# Change Log: CraigM, 9/14/2018, html render
+# ------------------------------------------------- #
+
+
+class Element:
+    """Base Class for HTML tags"""
+    def __init__(self, tag_name='element', content=None, **kwargs):
+        if content is None:
+            self.content = list()
+        else:
+            self.content = [content]
+        tag_style = ''
+        if 'style' in kwargs:
+            tag_style = ' style="{style}"'.format(style=kwargs['style'])
+        self.tag_open = "<{tag}{tag_style}>".format(tag=tag_name, tag_style=tag_style)
+        self.tag_close = "</{tag}>".format(tag=tag_name)
```

Comment:
no need for these to be class attributes; they can be local variables to the `render` method